engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
@sanjaypujare Please see my comment regarding attribute vs dt-site.xml setting. Why one is DAG level attribute and another a setting? Why is it OK to blindly overwrite STRAM_HTTP_CUSTOM_CONFIG attribute value if it was set when  CUSTOM_SSL_KEYSTORE_CONFIG is present? Can server vs client configuration be handled by a separate attribute/setting, but in a way that is less confusing to end users?